CHANGES.rst

```diff
@@ -38,6 +38,7 @@ astropy.io.fits
 - Check that the SIMPLE card is present when opening a file, to ensure that the
   file is a valid FITS file and raise a better error when opening a non FITS
   one. ``ignore_missing_simple`` can be used to skip this verification. [#10895]
+- Added ``append`` keyword to append table objects to an existing FITS file [#2632, #11149]
```

Comment:
Annoying nitpick, but this should have a blank line before the previous entry.  This can be fixed up just before merging though.